src/components/floaty-object.js

```diff
@@ -52,7 +52,7 @@ AFRAME.registerComponent("floaty-object", {
         this.bodyHelper.body.physicsBody.getAngularVelocity().length2() < angularThreshold * angularThreshold;
 
       if (isAtRest && isMine) {
-        this.el.setAttribute("body-helper", { type: "static" });
+        this.el.setAttribute("body-helper", { type: "kinematic" });
```

Comment:
We expect this change not to have a noticeable effect for end-users, but is more in line with our current understanding of the physics object state